index.js

```diff
@@ -8,19 +8,19 @@ const pTimes = require('p-times');
 const EventEmitter = require('events');
 
 class Queue extends EventEmitter {
-  constructor(mongoUrl, collectionName, waitDelay = 500, maxThreads = 1) {
+  constructor(mongoUrl, collectionName, waitDelay = 500, maxThreads = 1, db = null) {
```

Comment:
lets do a check on the mongoUrl, if string, need to connect, otherwise assume its a mongo db